pyproject.toml

```diff
@@ -52,8 +52,9 @@ inflection = ">=0.3.1"
 influxdb = "==5.0.0"
 logstash_formatter = "~0.5"
 mapbox = "==0.16.2"
-pyjwt = "~1.7"
+pint = "^0.9.0"
```

Comment:
Are you using Pint anywhere?

Reply:
Ah no, it was in there based off of the original plan for the telemetry data model. Will remove.